### ISSUE TITLE
C implementation allows `T` drawing command to reflect off `C` control point

### ISSUE THREAD

**Issue by Hixie** [google/iconvg#23]
If I'm reading the code correctly, in the C implementation, a 0xAF opcode (`C` command) followed by a 0x4F opcode (`T` command) will result in the `T` command using the last control point of the first opcode to compute the first control point of the second opcode, but the SVG spec says "if the previous command was not a Q, q, T or t, assume the control point is coincident with the current point".

**Comment by RazrFalcon** [comment]
Yes, looks like m, T, t and Q, q are implemented incorrectly. Here is a correctly working implementation from [resvg](https://github.com/RazrFalcon/resvg/blob/f6b00a233b43d54c3c853e58f8c97a60b58e7c7f/usvg/src/svgtree/parse.rs#L557).

I would even suggest sticking with just M, L, C, Z (only absolute) for simplicity.
